cryptography/src/lib.rs

```diff
@@ -142,6 +142,8 @@ pub trait BatchScheme {
     fn verify<R: RngCore + CryptoRng>(self, rng: &mut R) -> bool;
 }
 
+pub trait Digest: Array + Copy {}
```

Comment:
I had been trying to do something like this forever with a "trait alias" (which isn't supported on stable yet). This totally does the trick 🤦 .

Would you mind adding a comment above it?

Reply:
done